### PR TITLE
[WIP] Removes MAVProxy as a dependency.

### PR DESCRIPTION
**Pull request by tcr3dr**
This PR exists to maintain parity with master until all tests succeed.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -6,7 +6,7 @@
 
 [![Gitter](https://badges.gitter.im/Join%20Chat.svg)](https://gitter.im/diydrones/dronekit-python?utm_source=badge&utm_medium=badge&utm_campaign=pr-badge&utm_content=badge)
 
-This package contains the python language bindings for the drone API and the associated MAVProxy module.
+This package contains the python language bindings for the drone API.
```
Comment (mrpollo) [inline README.md]: This isn't relevant anymore, what @geeksville was referring to as "drone API" is DK-Cloud and the "MAVProxy module" is DK-Python, I nominate @hamishwillee to write a better description to what DK-Python is :sunglasses:

Comment (tcr3dr) [inline README.md]: Hah, was the key point here is removing MAVProxy references. I think a separate PR overhauling the droneapi -> dronekit references is in order.

Comment (mrpollo) [inline README.md]: agreed, lets add this to milestone v2

Comment (tcr3dr) [inline README.md]: Followed up here: https://github.com/dronekit/dronekit-python/issues/219

**Pull request closed without merging**